performance_tests/10zone1vav.idf

```diff
@@ -35,10 +35,12 @@
   Output:DebuggingData,0,0;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
+    1.,                      !- Name
```

Comment:
Interesting question.  Looks like these files aren't even running in the performance suite anymore.  I don't see 10zone1VAV and 20zone1VAV in the results list [here](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-d26f74b8a927051dad6e1810bcec27011a454872-x86_64-Linux-Ubuntu-14.04-gcc-4.8.html).  Oh ,that's because they are commented out in the [CMakeLists](https://github.com/NREL/EnergyPlus/blob/8bdce3a6ba876eea980ab77cd6b4ceea36bfc7fa/performance_tests/CMakeLists.txt#L10).  Looking at the history for 8.6 and 8.7 the only change to these files was incrementing the version number, so who knows *what* version these files really are.

So, the bigger question - if we aren't running these files (quite a few actually) then perhaps we should move them to an inactive folder?